src/azul/logging.py

```diff
@@ -163,20 +175,59 @@ def silenced_es_logger():
             assert es_log.level == original_log_level
 
 
-def http_body_log_message(body_type: str,
-                          body: bytes | bytearray | str | IO[bytes] | IO[str] | None,
+type BodyType = Literal['request', 'response']
+
+first_kibibit = 1024
+
+
+def http_body_log_message(body_type: BodyType,
+                          body: bytes | bytearray | str | IO[bytes] | IO[str] | JSON | None,
                           *,
-                          verbatim: bool = False,
+                          debug: int = 1,
```

Comment:
AFAICT, this is is always passed (so the default isn't used or needed), and, more importantly, the same value is passed (config.debug). I believe we discussed that we can eliminate the parameter exactly under the circumstances I am observing now.